### DISCUSSION TITLE
GitSync fails?

### DISCUSSION THREAD

**Discussion by SpeedKRNL** [GitbookIO/community#759] [3. Git Sync]
GitSync seems to be failing again
Anyone has the same problem?

**Comment by momentmaker** [comment]
same here

**Reply by aikoii** [reply]
Hey! Could you please share the IDs of your spaces? 🙏

**Reply by momentmaker** [reply]
`-Lag5F8SjQSN07r7gwvH`

**Reply by aikoii** [reply]
Thanks! We're looking into it 🙏

**Reply by aikoii** [reply]
Hey all! We have implemented a fix and are monitoring the situation. Thanks again for your patience with this 🌼
